### PR TITLE
[internal-fix] OSDOCS-10230: Fixed Networking doc callout errors on CP

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.14+

Issue:
[OSDOCS-10230](https://issues.redhat.com/browse/OSDOCS-10230)

Link to docs preview:
* [Egress router custom resource](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.html)
* [Deploying an egress router in redirect mode](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.html#nw-egress-router-redirect-mode-ovn_deploying-egress-router-ovn-redirection)
* [Configuring a service with MetalLB](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-configure-services.html#nw-metallb-configure-svc_configure-services-metallb)
* [Limit speaker pods to specific nodes](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-operator-install.html#nw-metallb-operator-limit-speaker-to-nodes_metallb-operator-install)
* [Troubleshooting BGP issues](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-troubleshoot-support.html#nw-metallb-troubleshoot-bgp_metallb-troubleshoot-support)
* [Adding the network attachment definition to your pods](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-hardware-offloading.html#adding-network-attachment-definition-to-pods_configuring-hardware-offloading)
* [Creating a network attachment definition](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-hardware-offloading.html#create-network-attachment-definition_configuring-hardware-offloading)
* [Configuring the SR-IOV network node policy](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-hardware-offloading.html#configure-sriov-node-policy_configuring-hardware-offloading)
* [Improving network traffic performance using a virtual function](https://75996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-hardware-offloading.html#improving-network-traffic-performance-using-vf_configuring-hardware-offloading)

Additional information:

### CODE REVIEW

**Comment by stevsmit** [comment]
LGTM

File: modules/nw-sriov-hwol-creating-sriov-policy.adoc [hunk 1]
```diff
@@ -43,9 +43,9 @@ spec:
   priority: 5
   resourceName: mlxnics
 ----
-<.> The name for the custom resource object.
-<.> Required. Hardware offloading is not supported with `vfio-pci`.
-<.> Required.
+<1> The name for the custom resource object.
+<2> Required. Hardware offloading is not supported with `vfio-pci`.
+<3> Required.
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/nw-sriov-hwol-creating-sriov-policy.adoc:48]: Correct. I was surprised when I seen it but finding the correct SME and so on might be another day's work. Thanks, Steve!

**Pull request merged**